packages/mgt-chat/src/utils/graph.ts

```diff
@@ -9,5 +9,7 @@ import { IGraph, Providers } from '@microsoft/mgt-element';
 
 export const graph = (component: string, version?: string): IGraph => {
   if (!component) throw new Error('Component name is required');
-  return Providers.globalProvider.graph.forComponent(component, version);
+  // NOTE: for some unknown reason, the global provider is not available on first
```

Comment:
Pretty sure that Provider.globalProvider should be initialized, but it's graph property is not.
This is because in Msal2Provider initializes the graph property inside the async initProvider method and after awaiting this._publicClientApplication.initialize() 

TLDR; there's a promise that hasn't completed when you're trying to access this if you don't restrict rendering based on auth state

Reply:
I moved that initialization around.